drake/examples/Valkyrie/@Valkyrie/Valkyrie.m

```diff
@@ -258,7 +260,7 @@
                                     'drake_instep_shift', 0.0,... % Distance to shift ZMP trajectory inward toward the instep from the center of the foot (m)
                                     'mu', 1.0,... % friction coefficient
                                     'constrain_full_foot_pose', true,... % whether to constrain the swing foot roll and pitch
-                                    'pelvis_height_above_foot_sole', 0.98,... % default pelvis height when walking
+                                    'pelvis_height_above_foot_sole', 1.01,... % default pelvis height when walking
```

Comment:
Yeah, it's just for Val, and it makes collision with the ankle limits less likely